vector-im/element-android
# [Device Manager] Render security recommendations (PSG-681)

## Type of change


## Content


Implement current session section as in the new layout.

## Motivation and context
Closes #6964



## Screenshots / GIFs




![security_recommendations](https://user-images.githubusercontent.com/1254401/187427682-107ee7ec-2427-4ee9-8ccc-88789dd0ff7e.png)


## Tests



- Enable the new device management feature flag
- Go to "Security & Privacy" settings
- Press the "Show all sessions (V2, WIP)" entry
- See the security recommendations are rendered as in the new layout

## Tested devices

- OS version(s):

## Checklist
